ci_tools/list_changed_files.py

```diff
@@ -13,7 +13,8 @@
 results = gdj(args.gitdiff)
 with open(args.result,'w', encoding='utf-8') as out:
     for file,changes in results.items():
-        for line in changes['addition']:
-            print(file, line, sep=' ', file=out)
-        for line in changes['deletion']:
-            print(file, line, sep=' ', file=out)
+        if file.startswith('pyccel/'):
+            for line in changes['addition']:
+                print(file, line, sep=' ', file=out)
+            for line in changes['deletion']:
+                print(file, line, sep=' ', file=out)
```

Comment:
Are you sure you want to check removed lines? Won't that make it impossible to get rid of errors?

Reply:
You are right! I need to think about this more carefully!